addons/main/CfgVehicles.hpp

```diff
@@ -98,6 +98,14 @@ class CfgVehicles {
             class AnyPerson2 : AnyPerson1 {};
         };
     };
+    class Buoy_base_F;
+    class GVAR(float): Buoy_base_F {
```

Comment:
Does `float` here actually work as intended (not as a var type but as a var name) ? O.o

Reply:
Is that protected? Macros would hit first and make it `bocr_main_float` before anything else sees it, right?
We can change it to `buoy` or something just to avoid confusion.

Reply:
I don't have the tiniest clue, which is why I asked. If it worked in your testing ...

Reply:
`float` will never hit the compiler. Pre-processor will replace GVAR(float) with `bocr_main_float` as previously mentioned before the compiler sees it.